e2e/testdata/helm-charts/ns-chart/templates/hpa.yaml

```diff
@@ -1,30 +0,0 @@
-{{- if .Values.autoscaling.enabled }}
```

Comment:
Not critical, but why's this removed?

Reply:
I'm struggling to find the related comment thread above, but I removed it so that we wouldn't have to grant the ns-reconciler permission to manage all resource types in the RepoSync test. Without this, that test can just grant permissions for core and app kinds.